default/stringtables/en.txt

```diff
@@ -2243,12 +2246,6 @@ MAP_SCALE_INDICATOR
 MAP_BTN_MENU
 Menu
 
-MAP_BTN_AUTO_ADVANCE_ENABLED
-Automatic Turn Advance
-
-MAP_BTN_MANUAL_TURN_ADVANCE
-Manual Turn Advance
-
```

Comment:
The removal of these entries that have just recently been added by Geoff is accidental I suppose? Pls fix.